Workflow/Modules/ModuleBase.py

```diff
@@ -357,7 +357,7 @@ def setApplicationStatus( self, status, sendFlag = True ):
       # The application status won't be updated in case the workflow or the step is failed already
       if not isinstance( status, basestring ):
         status = str( status )
-      self.log.verbose( 'setJobApplicationStatus(%d, %s)' % ( self.jobID, status ) )
+      self.log.verbose( 'setJobApplicationStatus(%s, %s)' % ( self.jobID, status ) )
```

Comment:
What is this then (seen in the certification):
2016-12-12 00:13:55 UTC dirac-jobexec   INFO: DIRAC JobID 108 is running at site LCG.CNAF.it
Executing StepInstance RunScriptStep1 of type ScriptStep1 ['ScriptStep1']
Exception while module execution
Module Script Script
%d format: a number is required, not str
== EXCEPTION ==
<type 'exceptions.TypeError'>: %d format: a number is required, not str

  File "/home/lhcb129/home_cr007_966679601/CREAM966679601/DIRAC_eQjSiYpilot/DIRAC/Core/Workflow/Step.py", line 268, in execute
    result = step_exec_modules[mod_inst_name].execute()

  File "/home/lhcb129/home_cr007_966679601/CREAM966679601/DIRAC_eQjSiYpilot/DIRAC/Workflow/Modules/ModuleBase.py", line 131, in execute
    self.setApplicationStatus( status )

  File "/home/lhcb129/home_cr007_966679601/CREAM966679601/DIRAC_eQjSiYpilot/DIRAC/Workflow/Modules/ModuleBase.py", line 360, in setApplicationStatus
    self.log.verbose( 'setJobApplicationStatus(%d, %s)' % ( self.jobID, status ) )